monitoring/README.md

```diff
@@ -72,10 +93,11 @@ This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyros
 
     a) WDS API Server: input into the following text boxes.
     - `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
-    - `SpaceName`: name of the KubeStellar space, e.g., *wds1*, *wds2*, etc.
+    - `APIService`: name of the service for the KS spaces APIserver, e.g., *vcluster*, *wds1*, *wds2*, etc.
 
         <img src="images/wds-apiserver-monitoring.png" width="60%" height="80%" title="wds-apiserver-monitoring"> 
 
+    Note: you can also monitor the APIserver of the hosting cluster using the following values: `SpaceNS: default` and `APIService: apiserver`
 
     b) KubeStellar controllers: input into the following text box and drop-down menu. 
 
```

Comment:
Are you sure that you want "kubestellar-controller-manager" and not "kubestellar-controller-manager-metrics-service"?

Reply:
Yes, indeed - good caught! that was typo.  In this PR, we are now we're using the kube-rbac-proxy to reach the kubestellar-controller. Hence, the correct service is the new updated one: ``kubestellar-controller-manager-metrics-service``